### DISCUSSION TITLE
Pimcore NotFoundHttpException with woff files

### DISCUSSION THREAD

**Discussion by MaxKumbe** [pimcore/pimcore#11129] [Q&A / Support]
Good afternoon, I loaded into the assets few files with extension .woff and .woff2.
When i try to load them with the corrisponding url that i copy from the pimcore asset info, I get this error: 
NotFoundHttpException 
No route found for "GET http://mysite.it:20081/imgfiles/fa-regular-400.woff2"
Why? The url is right, I have to add this particular extensions somewhere?
Thanks!

**Comment by zangab** [comment]
maybe u dont serve the public folder locally or at least check the public folder for the file. works without any problem on my side.
